docs/app/pages/components/components-sections/buttons/floating-action-button/floating-action-button.component.html

```diff
@@ -0,0 +1,85 @@
+<ux-floating-action-buttons [ngClass]="direction" [direction]="direction">
+
+    <ux-floating-action-button fab-primary icon="hpe-contract" tooltip="Show Actions" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-add" tooltip="Add New Item" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-analytics" tooltip="Show Analytics" container="body" [placement]="placement"></ux-floating-action-button>
+    <ux-floating-action-button icon="hpe-app" tooltip="Show Overview" container="body" [placement]="placement"></ux-floating-action-button>
+
+</ux-floating-action-buttons>
+
+<hr>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-panel heading="Customize Example...">
+                <div class="row">
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="top">Top</ux-radio-button>
+                    </div>
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="right">Right</ux-radio-button>
+                    </div>
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="bottom">Bottom</ux-radio-button>
+                    </div>
+                    <div class="col-md-3">
+                        <ux-radio-button [(ngModel)]="direction" option="left">Left</ux-radio-button>
+                    </div>
+                </div>
+            </accordion-panel>
+        </accordion>
+    </div>
+</div>
+
+<p>
+    Floating Action Buttons are contained in a <code>ux-floating-action-buttons</code> component. 
+    Each action button should be added as child using the <code>ux-floating-action-button</code> component.
```

Comment:
Since they aren't explicitly `button`s, and the example doesn't show it, maybe mention that the action should be attached via a `(click)` attribute.